docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -119,6 +119,15 @@ If you are searching for a well tested solution, with a large community and nice
 
 Learn more about it on the dedicated [Nuxt](./nuxt.md) page.
 
+## Servue
+[Servue]https://futureaus.github.io/servue/ shares a similar scope to Nuxtjs, but with more freedom and targetted more towards Multi-page-applications using existing server/routing systems such as koa/koa-router and express.js
+
+It’s this easy to render .vue files:
+
+```js
```

Comment:
We don't really show code examples in other places on the page, as it's not relevant for some solutions. You could add a link to a tutorial if you like, using the useful-link component.